deprecation.py

```diff
@@ -42,7 +43,8 @@ class DeprecatedWarning(DeprecationWarning):
 
     :param function: The function being deprecated.
     :param deprecated_in: The version that ``function`` is deprecated in
-    :param removed_in: The version that ``function`` gets removed in
+    :param removed_in: "The version that function gets removed in, or
```

Comment:
Those double quotes end up getting rendered in the docs output so they should be removed, and "function" should continue to be wrapped in double backticks. I think a better wording of this might be something like:

```
The version or :class:`datetime.date` specifying when ``function`` gets removed.
```

The docs build (from `tox -e docs`) uses "intersphinx" and is connected to `docs.python.org`, so using the `:class:` prefix on `datetime.date` in that line allows our docs to link straight to the standard library docs for that type name.

It ends up rendering like this:

> The version or [datetime.date](https://docs.python.org/dev/library/datetime.html#datetime.date) specifying when `function` gets removed.

Reply:
I have made the change. I was unaware of how the docs are built (it's pretty cool).

Reply:
The changes have been made. I was unaware of how the documentation works (It's pretty nice)